pom.xml

```diff
@@ -78,11 +78,11 @@
   <properties>
     <connid.version>1.5.0.2</connid.version>
     
-    <azure.version>1.7.0</azure.version>
-    <azure.adal4j.version>1.4.0</azure.adal4j.version>
-    <azure.client.authentication>1.3.0</azure.client.authentication>
+    <microsoft.graph.version>5.8.0</microsoft.graph.version>
+    <azure.identity.version>1.4.2</azure.identity.version>
+    <azure.msal4j.version>1.11.0</azure.msal4j.version>
+    <junit.version>5.7.1</junit.version>
```

Comment:
Please remove the JUnit version and restore the test dependency declaration to avoid errors.